scripts/checklistMove.js

```diff
@@ -0,0 +1,23 @@
+$(".get").on("click",function(){
```

Comment:
Add a comment for anonymous function. 

For PEAR standards brackets should be next line for function